app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/ArticleSearchResultItem.vue

```diff
@@ -79,54 +90,27 @@ export default {
     },
     locale: {
       type: String,
-      default: 'en-US',
+      default: '',
+    },
+  },
+  computed: {
+    plainBody() {
+      return this.getPlainText(this.body);
     },
   },
   methods: {
-    handleClick() {
-      this.$emit('click');
+    handleInsert(e) {
+      e.preventDefault();
```

Comment:
Why did we add a prevent here?

Reply:
Without the preventDefault the event wasn't emitting.